server/bundler/loader.js

```diff
@@ -128,8 +127,8 @@ function splitTemplate( path, section ) {
 		'		return;',
 		'	}',
 		'	context.store.dispatch( { type: "SECTION_SET", isLoading: true } );',
-		'	preload( ' + moduleString + ' )',
-		'		.then( sectionModule => setSection( context, ' + sectionString + ', sectionModule ));',
+		'	preload( ' + JSON.stringify( section.name ) + ' )',
+		'		.then( sectionModule => { setSection( context, ' + sectionString + ', sectionModule ); next() } );',
```

Comment:
We should check if this code does not go through babel in which case we should not use fat arrow functions

Reply:
IMO it does